examples/verify_tsqrtP_calibration.py

```diff
@@ -0,0 +1,148 @@
+from itertools import cycle
+
+colorcyc_list = plt.rcParams["axes.prop_cycle"].by_key()["color"]
+color_cycle = cycle(
+    colorcyc_list
+)  # this can be done more than once to spin up multiple lists
+
+V_atten_ratio = 102.2  # attenutation ratio
+skip_plots = 33  # diagnostic -- set this to None, and there will be no plots
+slicewidth = 2e6
+
+
+with psd.figlist_var() as fl:
+    for filename, nodename in [
+        (
+            "240819_test_amp0p05_calib_pulse_calib.h5",
+            "pulse_calib_5",
+        ),  # low power
+        (
+            "240819_amp0p1_calib_pulse_calib.h5",
+            "pulse_calib_2",
+        ),  # low power
+        (
+            "240819_amp0p2_calib_repeat_pulse_calib.h5",
+            "pulse_calib_9",
+        ),  # low power
+    ]:
+        d = psd.find_file(
+            filename, expno=nodename, exp_type="ODNP_NMR_comp/test_equipment"
+        )
+        assert (
+            d.get_prop("postproc_type") is not None
+        ), "No postproc type was set upon acquisition"
+        amplitude = d.get_prop("acq_params")["amplitude"]
+        fl.basename = f"amplitude = {amplitude}"
+        if not d.get_units("t") == "s":
+            print(
+                "units weren't set for the t axis or else I can't read them from the hdf5 file!"
+            )
+            d.set_units("t", "s")
+        d *= V_atten_ratio
+        d /= np.sqrt(50)  # V/sqrt(R) = sqrt(P)
+        pulse_lengths = d.get_prop("programmed_t_pulse_us")
+
+        def switch_to_plot(d, j):
+            thislen = pulse_lengths[j]
+            fl.next(f"pulse length = {thislen}")
+
+        def indiv_plots(d, thislabel, thiscolor):
+            if skip_plots is None:
+                return
+            for j in range(len(d["beta"])):
+                if j % skip_plots == 0:
+                    switch_to_plot(d, j)
+                    fl.plot(
+                        d["beta", j],
+                        alpha=0.2,
+                        color=thiscolor,
+                        label=thislabel,
+                    )
+
+        indiv_plots(d, "raw", "blue")
+        # {{{ data is already analytic, and downsampled to below 24 MHz
+        indiv_plots(abs(d), "analytic", "orange")
+        d.ft("t")
+        # {{{ apply frequency filter
+        d.ift("t")
+        SW = 1 / (d["t"][1] - d["t"][0])  # nyquists freq
+        carrier = (
+            d.get_prop("acq_params")["carrierFreq_MHz"] * 1e6
+        )  # signal frequency
+        n = np.floor(
+            (carrier + SW / 2) / SW
+        )  # nearest integer multiple of sampling frequency
+        nu_a = carrier - n * SW  # find the aliasing frequency
+        if nu_a < 0:
+            perceived_nu = abs(
+                nu_a
+            )  # we removed the negative frequencies so it's counterpart is the abs
+            d.run(np.conj)
+        else:  # otherwise subtract the difference between the signal and nyquist from nyquist's
+            perceived_nu = SW - nu_a
```

Comment:
Same comments, as in other PR's about the fact that you have two variables that appear to be filling the same role and that the absolute value is not actually explained.  Based on comments you provide in the code here, it should be possible to sketch yourself a figure of the aliasing and see why it's justified to simply take the absolute value here, if in fact, it is justified.  


I would suggest hashing this out **here** since this is the simplest poll request that involves this issue

Reply:
handled with new funciton